src/sentry/eventstore/models.py

```diff
@@ -210,7 +210,10 @@ def get_minimal_user(self) -> User:
                 {"id": user_id, "email": email, "username": username, "ip_address": ip_address}
             )
 
-        return self.get_interface("user")
+        user = self.get_interface("user")
+        if user is not None:
+            user._data.pop("sentry_user", None)
+            return user
```

Comment:
Fixes the test `tests/sentry/eventstore/test_models.py::EventTest::test_snuba_data`.

`get_minimal_user` intends to extract the _minimal_ user that may be exposed to external services, so there's no reason to expose `sentry_user`. Alternatively, we could expose the value from Snuba, but I see no reason to do that as the field was originally added for DDM.

Reply:
Done in [071238f](https://github.com/getsentry/sentry/pull/66401/commits/071238f3ab6bd78283e0d2bcac153ade30e9c30c).